### PR TITLE
plpgsql `NULL` and `RETURN` STATEMENTS

### PR DESCRIPTION
**Pull request by taroface**
DOC-9847
DOC-9843

Document `NULL` and `RETURN` (with no expression) statements in PL/pgSQL.

### CODE REVIEW

File: src/current/v24.1/plpgsql.md [hunk 1]
```diff
@@ -254,7 +261,9 @@ WHILE condition LOOP
 
 For an example, see [Create a stored procedure that uses a `WHILE` loop]({% link {{ page.version.version }}/create-procedure.md %}#create-a-stored-procedure-that-uses-a-while-loop).
 
-### `EXIT` and `CONTINUE` statements
+### Add exit and continue logic
```
Comment (DrewKimball) [inline src/current/v24.1/plpgsql.md:264]: [nit] This heading seems a little odd when we have a RETURN section in between the EXIT and CONTINUE examples.

**Review by DrewKimball — APPROVED** [review]
LGTM

File: src/current/v24.1/plpgsql.md (hunk 1 above)
Comment (taroface) [reply src/current/v24.1/plpgsql.md]: How does "Control execution flow" sound to you?

Comment (DrewKimball) [reply src/current/v24.1/plpgsql.md]: That SGTM.

File: src/current/v24.1/create-function.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ If you grant `EXECUTE` privilege as a default privilege at the database level, n
 Parameter | Description
 ----------|------------
 `routine_create_name` | The name of the function.
-`routine_param` | A comma-separated list of function parameters.
+`routine_param` | A comma-separated list of function parameters, specifying the mode, name, and type.
```
Comment (mdlinville) [inline src/current/v24.1/create-function.md:34]: Does this mean that you set `routine_param` to a series of mode:name:type groupings? Is there an example anywhere?

**Review by mdlinville — APPROVED** [review]

**Pull request merged**